cmake/QualityControlConfig.cmake.in

```diff
@@ -0,0 +1,19 @@
+@PACKAGE_INIT@
+
+set(QualityControl_VERSION @PROJECT_VERSION@)
+
+get_filename_component(QualityControl_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+include(CMakeFindDependencyMacro)
+
+list(APPEND CMAKE_MODULE_PATH ${QualityControl_CMAKE_DIR})
+find_dependency(Boost)
+
+list(REMOVE_AT CMAKE_MODULE_PATH -1)
+
+if(NOT TARGET QualityControl::QualityControl)
```

Comment:
TBD whether `AliceO2::QualityControl` or `QualityControl::QualityControl` (WP3 I guess)

Reply:
I am in favour of AliceO2::QualityControl, I will change it